spec/system/task_spec.rb

```diff
@@ -1,27 +1,36 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
```

Comment:
associationが設定されていれば、この指定は要らないような気がします。
```suggestion
  let(:task) { create(:task) }
```

Reply:
fc22678c073dd0bc67f65b42c53fd26e50fc7c78 修正しました。